### PR TITLE
Split notification settings

### PR DESCRIPTION
**Pull request by F2had**
Split notifications by type (Iqama/Athan) rather than by prayer. Which hopefully will make it easier to update settings for Iqama/Athan.

### CODE REVIEW

File: lib/modules/settings/screens/notification.dart [hunk 1]
```diff
@@ -47,41 +49,41 @@ class _NotificationScreenState extends State<NotificationScreen> {
...
       settingsSection = SettingsSection(
-        title: Text(t!.prayersAlerts),
+        title: Text(t.prayersAlerts),
         tiles: [
-          _buildPrayerSettingsTile(PrayerType.fajr, t.fajr),
-          _buildPrayerSettingsTile(PrayerType.duhur, t.duhur),
-          _buildPrayerSettingsTile(PrayerType.asr, t.asr),
-          _buildPrayerSettingsTile(PrayerType.maghrib, t.maghrib),
-          _buildPrayerSettingsTile(PrayerType.isha, t.isha),
+          _buildSettingsTile(
```
Comment (Amjad50) [inline lib/modules/settings/screens/notification.dart:78]: maybe comment this since its not finished

**Review by Amjad50 — COMMENTED** [review]
I'm also thinking, if we enable pre-athan notification, we should be able to disable the athan itself.
most ppl don't want another setting and just a pre-athan timer just to remind them

File: lib/modules/settings/screens/notification.dart (hunk 1 above)
Comment (F2had) [reply lib/modules/settings/screens/notification.dart:78]: Actually I was planning to start working on it right after this.

**Comment by F2had** [comment]
> I'm also thinking, if we enable pre-athan notification, we should be able to disable the athan itself.
> 
> most ppl don't want another setting and just a pre-athan timer just to remind them

Seems reasonable, however should we add ability to disable in this pr or after implementing the pre athan?

File: lib/modules/settings/screens/notification.dart (hunk 1 above)
Comment (Amjad50) [reply lib/modules/settings/screens/notification.dart:78]: Ah ok, I thought it was ready for review

Comment (F2had) [reply lib/modules/settings/screens/notification.dart:78]: I mean in a different pr

**Comment by Amjad50** [comment]
> should we add ability to disable in this pr or after implementing the pre athan?

After, together with pre-athan

**Review by Amjad50 — APPROVED** [review]

**Pull request merged**